src/pages/Patch.tsx

```diff
@@ -42,13 +43,15 @@ const PatchCore: React.FC = () => {
       ),
   });
   useEffect(() => stopPolling, [stopPolling]);
+
   const patch = get(data, "patch");
   const status = get(patch, "status");
   const description = get(patch, "description");
   const activated = get(patch, "activated");
   if (activated === false) {
     router.replace(`${paths.patch}/${id}/configure`);
   }
+  usePageTitle(`Patch - ${description} - ${patch && patch.patchNumber}`);
```

Comment:
I don't think it makes sense to include the description since it can be quite long. Just patch # seems fine